README.md

```diff
@@ -75,24 +75,24 @@ The ecosystem and software Redox OS provides is listed below.
 
 ## <a name="compile-help"> Help! Redox won't compile! </a>
 
-Sometimes things go wrong when compiling. Try the following before opening an issue:
+If you encounter an issue during the compilation, please try the following before opening an issue:
 
 1. Run `make clean`.
 2. Run `git clean -Xfd`.
-3. Make sure you have **the latest version of Rust nightly!** ([rustup.rs](https://www.rustup.rs) is recommended for managing Rust versions. If you already have it, run `rustup`).
+3. Ensure you are using **the latest version of Rust nightly!** ([rustup.rs](https://www.rustup.rs) is recommended for managing Rust versions. If it is installed, run `rustup`).
 4. Update **GNU Make**, **NASM** and **QEMU/VirtualBox**.
 5. Pull the upstream master branch (`git remote add upstream git@github.com:redox-os/redox.git; git pull upstream master`).
 6. Update submodules (`git submodule update --recursive --init`).
 
-and then rebuild!
+and rebuild!
 
 ## <a name="contributing"> Contributing to Redox </a>
 
-If you're interested in this project, and you'd like to help us out, [here](CONTRIBUTING.md) is a list of ways you can do just that.
+If you're interested in this project, and would like to help, [here](CONTRIBUTING.md) are a few things you can do.
 
 ## <a name="cloning-building-running"> Cloning, Building, and Running </a>
 
-Redox is big, even compressed. Downloading the full history may take a lot of bandwidth, and can even be costly on some data plans. Clone at your own risk!
+Redox is a large operating system, even when it is compressed. Downloading the full history will consume bandwidth, and may cost extra on some data plans. Please check the allowed usage before you clone.
```

Comment:
"Redox is big" -> true,

"Redox is a big operating system" -> false,

Any operating system is gonna be big, but for an operating system, Redox is tiny. The original text better reflects reality.